File: Coding Exercises/TypeScript/Day60/reverseNumber.ts
function reverseNumber(num: number): number {
  let reversedNum: number = 0;
  const sign: number = (num >= 0) ? 1 : -1;
  num = Math.abs(num);

  while (num !== 0) {
      const lastDigit: number = num % 10;
      reversedNum = reversedNum * 10 + lastDigit;
      num = Math.floor(num / 10);
  }

  return reversedNum * sign;
}

File: Coding Exercises/Python/Day60/reverse_number.py
def reverse_number(num):
    reversed_num = 0
    sign = 1 if num >= 0 else -1
    num = abs(num)

    while num != 0:
        last_digit = num % 10
        reversed_num = reversed_num * 10 + last_digit
        num = num // 10

    return reversed_num * sign
